test/CMakeLists.txt

```diff
@@ -44,7 +44,11 @@ function(add_fmt_test name)
   endif ()
   add_fmt_executable(${name} ${sources})
   target_link_libraries(${name} ${libs})
-
+  # use C++20 if C++20 is available for https://github.com/fmtlib/fmt/pull/4042
+  get_property(cxx_features GLOBAL PROPERTY CMAKE_CXX_KNOWN_FEATURES)
+  if(cxx_std_20 IN_LIST cxx_features)
+    target_compile_features(${name} PRIVATE cxx_std_20)
+  endif()
```

Comment:
This standard should be controlled externally, not here. Let's revert.

Reply:
@vitaut 
Sorry I try to revert both cxx_std and .gradle files.

Reply:
@vitaut 
Now I finished.
Cloud you review it again?